examples/mcp9808_temperature_limits.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2021 Jose David M.
+# SPDX-License-Identifier: MIT
+
+"""
+Show the MCP9808 to setup different temperature values
+"""
+
+import time
+import board
+import adafruit_mcp9808
+
+i2c = board.I2C()  # uses board.SCL and board.SDA
+mcp = adafruit_mcp9808.MCP9808(i2c)
+
+# Change the values according to the desired values
+print("Setting Temperature Limits")
+mcp.upper_temperature = 23
+mcp.lower_temperature = 10
+mcp.critical_temperature = 100
+
+# To verify the limits we need to read the temperature value
+print(mcp.temperature)
+time.sleep(0.3)  # This is the time temperature conversion at maximum resolution
+
+# Showing temperature Limits
+print("Lower Temperature Limit:", mcp.below_lt)
+print("Upper Temperature Limit:", mcp.above_ut)
+print("Critical Temperature Limit:", mcp.above_ct)
```

Comment:
Shouldn't these be the properties instead of the registers directly? I'm confused about the format differences. I assume you'd need only one set.

Reply:
Thanks for reviewing, 
We have the properties and the registers because,
The registers are the flags, and we get 1 or 0 when the temperatures are reached.  the register address is used verify conditions.
However you can set these temperature on whatever you want, and are not related to these flags, that is why there are two sets. one gives True or False, and the other gives the value. there are located in two completely different registers. The True or False is located in the Temperature Address  (0x05). bits 7-5
And the values are located in registers (0x04, 0x03, and 0x02).

I could ask a property to them, no problem just did not want it to bloat the library and the have problems to load in small boards. 
I could create the properties, no problem at all, Guidance is needed here :) 
Thanks..

Reply:
Ah, these would be clearer if you do an `if`:

```suggestion
# Showing temperature Limits
if mcp.below_lt:
    print("too cold!")
if mcp.above_ut:
    print("getting hot!")
if mcp.above_ct:
    print("Above critical temp!")
```

I'd also like to see them written out `above_critical_temperature` for example.

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_MCP9808/pull/29/commits/693758d28353cfd8617a2a516385b9d046044971